setup/README.md

```diff
@@ -3,78 +3,70 @@
-    *   Are you using GitHub? (y/n)
-        *   If you choose yes, the GitHub specific Pub/Sub topic, subscriptions, and worker will be created.
-    *   BigQuery setup
-        *   If you've never setup BigQuery before, a setup page will open in your browser.
-    *   Would you like to create a separate new project to test deployments for the four key metrics? (y/n)
-        *   You have the option of creating a new Google Cloud project to test out doing deployments and seeing how they are tracked in the dashboard.  However, if you already have a project with deployments, you may select no to skip this step.  You do not need to select yes to generate mock data.
-    *   Would you like to generate mock data? (y/n)
-        *   If you select yes, a script will run through and send mock GitLab or GitHub events to your event-handler.  This will populate your dashboard with mock data.  The mock data will include the work "mock" in the source. You can generate mock data without using the setup script. See [Generating mock data](../readme.md).
+    *   Would you like to create a new Google Cloud Project for the four key metrics? (y/N)
+        * If you choose no, you will be asked later to provide info about the project you wish to use.
+    *   Choose the event sources to configure...
+        *   Which version control system are you using?
+            * Choose the appropriate option for your VCS, or choose "other" to skip VCS integration
+        * Which CI/CD system are you using?
+            * Choose the appropriate option for your CICD system, or choose "other" to skip CICD integration
+        * _(see `/README.md#extending-to-other-event-sources` to integrate event sources not available during setup)_
+    *   Would you like to generate mock data? (y/N)
+        *   If you select yes, a script will run through and send mock GitLab or GitHub events to your event-handler.  This will populate your dashboard with mock data.  The mock data will include the work "mock" in the source. You can generate mock data without using the setup script. See [Generating mock data](../readme.md). 
+            *   To exclude the mock data from the dashboard, update the SQL script to filter out any source with the word mock in it by adding: `WHERE source not like "%mock"`.
+
+### Making changes
+At some point after running the setup script, you may want to make modifications to your infrastructure. Or, the Four Keys project itself may be updated with a new configuration. Terraform can be used to apply any incremental changes: after updating the configuration files, run `terraform apply`. You'll be prompted to confirm the planned changes; review them carefully, then type `yes` to proceed.
+> Tip: To make changes that will apply to your infrastructure, without editing the core configuration, consider using [Terraform Override Files](https://www.terraform.io/docs/language/files/override.html).
```

Comment:
Can you add some kind of warning about making changes outside of Terraform?  Maybe something like:  "If you make changes to the project outside of Terraform, they will not be tracked and cannot be managed by Terraform.  This includes pub/sub topics, subscriptions, permissions, service accounts, services, etc."

Reply:
Good call -- added!